atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -208,6 +209,36 @@ def build_list(self, manifests):
                 ],
         }, indent=4)
 
+    def check_existing_vr_tag(self):
+        self.primary_images = get_primary_images(self.workflow)
```

Comment:
Is there any reason for setting this instance var here instead of keeping it within this method? i.e., why do we need `self` here?

Reply:
no, I can just use local var